package.json

```diff
@@ -15,13 +15,12 @@
     "deploy:docs": "gh-pages -d docs/build -m 'deploy docs [ci skip]'",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
-    "lint:watch": "watch 'npm run lint' docs gulp src test",
+    "lint:watch": "watch 'npm run lint --silent' docs gulp src test",
     "prepublish": "npm run build",
     "release:major": "ta-script npm/release.sh major",
     "release:minor": "ta-script npm/release.sh minor",
     "release:patch": "ta-script npm/release.sh patch",
     "start": "gulp",
-    "pretest": "npm run lint --silent",
```

Comment:
Removed lint on pretest, otherwise you need to fix all lint errors before you can run `npm run test:watch` because the lint exits before the test watcher can start.

The test watcher runs lint, as well as CI on pre test.